### ISSUE TITLE
Use UpperCase in Search Hash Header

### ISSUE THREAD

**Issue by hotail** [saferwall/saferwall#242]
Hello LordNoteworthy, 

Some my Hash Program return Upper Case output. I just want to share with you that i can edit it in /ui/src/components/partials/Header.vue 

from "this.hash" to "this.hash.toLowerCase()"

```js
searchByHash() {
      if (!this.hash.trim()) {
        this.$awn.warning("Empty Field!")
        return
      }
      this.$http
        .get(`${this.$api_endpoints.FILES}${this.hash}/`, {
          validateStatus: (status) => status === 200,
        })
        .then((data) => {
          this.updateHash(this.hash)
          this.track()
          this.$router.push(this.$routes.SUMMARY.path + this.hash)
        })
        .catch(() => {
          this.$awn.alert(
            "Sorry, we couldn't find the file you were looking for, please upload it to view the results!",
          )
        })
    },
    track() {
      this.$gtag.event("search", {
        search_term: this.hash,
      })
    },
```

**Comment by LordNoteworthy** [comment]
Hello @hotail !

Thanks, good catch, I missed that 👍 

It is better to fix this one on the backend side, because if someone is using the web apis directly, he would fall into the same issue.

I deployed a fix on: c6d42b2af10b78365bfc8ebec65d562a4f3e0dca and I upgraded the web docker image to `0.0.3`. You can pull it and retry.

Let me know if that works.

**Comment by hotail** [comment]
It works, thanks @LordNoteworthy
